modules/mono/editor/bindings_generator.cpp

```diff
@@ -523,7 +523,27 @@ void BindingsGenerator::_append_xml_method(StringBuilder &p_xml_output, const Ty
 				p_xml_output.append(p_target_itype->proxy_name);
 				p_xml_output.append(".");
 				p_xml_output.append(target_imethod->proxy_name);
-				p_xml_output.append("\"/>");
+				p_xml_output.append("(");
+				bool first_key = true;
+				for (const ArgumentInterface &iarg : target_imethod->arguments) {
+					const TypeInterface *arg_type = _get_type_or_null(iarg.type);
+
+					ERR_FAIL_NULL(arg_type); // Argument type not found
+					if (first_key) {
+						first_key = false;
+					} else {
+						p_xml_output.append(", ");
+					}
+					if (iarg.def_param_mode == ArgumentInterface::NULLABLE_VAL) {
+						p_xml_output.append("Nullable{");
+					}
+					String arg_cs_type = arg_type->cs_type + _get_generic_type_parameters(*arg_type, iarg.type.generic_type_parameters);
+					p_xml_output.append(arg_cs_type.replacen("<", "{").replacen(">", "}").replacen("params ", ""));
```

Comment:
Is the `params` replace needed? If so, is this a bug in `_get_generic_type_parameters`?

Reply:
It's not a bug in `_get_generic_type_parameters`, but a consequence of how the documentation handles parameters; it doesn't accept `params ` as a valid prefix

eg: `public static void Function<T>(params object[] args)` in a docstring would be explicitly referred to with `Function{T}(object[])`, but trying `Function{T}(param object[])` would fail

Reply:
Oh wait, I see it now, it's `arg_type->cs_type` that may contain the `params ` prefix. I think we should probably be using `arg_type->proxy_type`.

Reply:
Just checked, and it indeed needs to be fully qualified